Facile/src/main/java/at/pollaknet/api/facile/renderer/ILAsmRenderer.java

```diff
@@ -932,13 +932,17 @@ private String render(MethodBody methodBody, DebugInformation debugInformation)
 		buffer.append(methodBody.getCodeSize());
 		buffer.append(" byte");
 		
-		if(	methodBody.getMethodToken() == reflector.getCliHeader().getEntryPointToken() &&
-			!ByteReader.testFlags(	reflector.getCliHeader().getFlags(),
-									CliHeader.FLAGS_NATIVE_ENTRY_POINT		)				) {
-			buffer.append(newLine);
-			buffer.append(".entrypoint ");
+
+		if(reflector != null){
+			if(	methodBody.getMethodToken() == reflector.getCliHeader().getEntryPointToken() &&

```

Comment:
And here

Reply:
Maybe we can add a parameter to this method for the entry point token and make the method public